arch/arm/soc/st_stm32/stm32f1/Kconfig.series

```diff
@@ -12,6 +12,7 @@ config SOC_SERIES_STM32F1X
 	select SOC_FAMILY_STM32
 	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
 	select HAS_STM32CUBE
+	select CPU_HAS_MPU
```

Comment:
Hence, we consider CPU_HAS_MPU will be selected
- at series level if we consider all SoCs of a serie support MPU (F4 and L4 for now)
- at SoC level if, inside one series, some SoC support MPU, and some other don't
I'm alined. Though, it'd be nice to store this information somewhere (help section of config STM32_ARM_MPU_ENABLE config flag maybe?)

Otherwise LGTM